### ISSUE TITLE
Update Understanding Success Criteria 1.4.8 Visual Presentation to clarify the purpose of the phrase "(aligned to both the left and the right margins)"

### ISSUE THREAD

**Issue by daniel-montalvo** [w3c/wcag#5323]
Based on comments received by the Japan delegation in the context of the WCAG 2.2 2024 PAS submission to ISO, we have been requested to amend SC 1.4.8 Visual Presentation. The most likely solution based on the position of the Working Group about updating WCAG 2.2's normative text seems to qualify the purpose of the phrase "(aligned to both the left and the right margins)" to clarify that this SC can also be applied to vertical writing.

**Comment by patrickhlauke** [comment]
keeping this in spec language, the logical (hah) candidates would be `inline-start` and `inline-end`. problem is, no mere mortal likely knows what these actually mean https://developer.mozilla.org/en-US/docs/Glossary/Flow_relative_values#inline_direction

**Comment by patrickhlauke** [comment]
also, assuming this would be either be an erratum for 2.2, or set aside for the potential 2.3 version?

**Comment by bruce-usab** [comment]
The purpose of the parenthetical is to explain “justified”.
1. Is there a better typographical term we might use?  Or a term that translates easier?
1. Is the addition of white space in vertical writing (to keep both top and bottom margins looking neat and tidy) a common practice?  (Full justification is common with western newspapers and books.)
1. If it is, does that additional white space interfere with fluid reading (as it frequently does with western text)?

If either (2) or (3) is no, then Note 2 applies and that particular bullet (in the SC) should be ignored for vertical writing systems (and no normative change is necessary).

**Comment by iadawn** [comment]
@murata2makoto Would you have some insight you can provide to help answer questions 2 and 3 from Bruce?

Thanks
